frontend/src/utils.js

```diff
@@ -104,9 +104,29 @@ function copyToClipboard(str) {
   return result;
 }
 
+const LOCALIZE_NUMBERS = !!(
+  typeof Intl === 'object' &&
+  Intl &&
+  typeof Intl.NumberFormat === 'function'
+);
+
+const UNITS = ['B', 'kB', 'MB', 'GB'];
+function bytes(num) {
+  const exponent = Math.min(Math.floor(Math.log10(num) / 3), UNITS.length - 1);
+  const n = Number(num / Math.pow(1000, exponent));
+  const nStr = LOCALIZE_NUMBERS
+    ? n.toLocaleString(navigator.language, {
+        minimumFractionDigits: 1,
+        maximumFractionDigits: 1
+      })
```

Comment:
Sorry, I've always struggled to understand localization...
Not sure I understand how this is working... What happens here if I have a local `navigator.language` of some non-supported localization on stage/prod (like Hebrew, for example), but I get a match on my second or third choice l20n fallback (such as Japanese)? Would the numbers be displayed in my preferred Hebrew language format, but the surrounding text be Japanese because that is one of the official production `availableLanguages`? Or am I totally missing something?